libs/net/dal/Services/ContentService.cs

```diff
@@ -501,5 +501,22 @@ public IEnumerable<NotificationInstance> GetNotificationsFor(long contentId)
         return this.Context.NotificationInstances
             .Where(n => n.ContentId == contentId);
     }
+
+    /// <summary>
+    /// Update the specified 'entity' in the database.
+    /// </summary>
+    /// <param name="updated"></param>
+    /// <returns></returns>
+    /// <exception cref="InvalidOperationException"></exception>
+    /// TODO: Switch to not found exception throughout services.
+    public Content UpdateStatusOnly(Content updated)
+    {
+        var original = FindById(updated.Id) ?? throw new InvalidOperationException("Entity does not exist");
+
+        this.Context.Entry(original.Status).CurrentValues.SetValues(updated.Status);
+        this.Context.ResetVersion(original);
```

Comment:
Is the desired behavior to ignore concurrency?  This will result in this operation to always 'win', essentially ignoring any changes that were made by other processes.  Which could result in loss of data.

Reply:
removed `ResetVersion()` call as per discussion